packages/cli-kit/src/node/cli.ts

```diff
@@ -86,12 +89,14 @@ const reportError = async (errorToReport: Error): Promise<Error> => {
   // eslint-disable-next-line no-prototype-builtins
   if (Error.prototype.isPrototypeOf(errorToReport)) {
     mappedError = new Error(errorToReport.message)
-    if (errorToReport.stack) {
-      // For mac/linux, remove `file:///` from stacktrace
-      // For windows, remove `file:///C:/` from stacktrace
-      const regex = '\\((.*node_modules.)(@shopify.)?'
-      mappedError.stack = errorToReport.stack.replace(new RegExp(regex, 'g'), '(')
-    }
+    const mappedStacktrace = new StackTracey(errorToReport)
+      .clean()
+      .items.map((item) => {
+        const filePath = normalize(item.file).replace('file:/', '/').replace('C:/', '')
```

Comment:
I replaced part of the regex expression with the [normalize](https://nodejs.org/api/path.html#pathnormalizepath) function because that's intended to be used for fixing these normalizations issue. Then I debated between doing the replacements with regex or string replacement and I found the replaces would be easier to follow when reading the code. 
I'm in contact with Bugsnag because they are only able to relativize the paths of macOS stacktraces but not Windows'. I think tried with and without the `C:/` prefix but it doesn't work.